### PR TITLE
Add Styling to match Zeplin Design

### PR DESCRIPTION
**Pull request by DevaWinner**
In this Pull Request, I have made the project's design in line with specifications. I have added any missing JSX elements necessary to match the desired design without introducing extra functionality. Additionally, I applied the CSS rules provided in Zeplin to ensure that the website adheres to the exact styling specifications.

##Changes Made:

- [x] Added missing JSX elements to achieve the same structure and layout as per Zeplin's design.
- [x] Utilized Zeplin's CSS rules to style the components and elements, ensuring they match the specified design. 

Please review the changes and let me know if there are any further adjustments needed or if you have any feedback. 

Thank you!

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+import './styles/book.css';
 
 function Book({ book, deleteBook }) {
   return (
-    <div>
-      <li>
-        <div>
-          <p>Title:</p>
-          {book.title}
+    <li className="book-container">
```
Comment (tufoinnkuo10) [inline src/components/Book.js:8]: - [ ] nice job so far... please kindly ensure your design is identical to the zeplin design as much as possible. please the 
- `comment`, `edit` and `remove` should be aligned on the same line like in the original design
- also, kindly ensure the `current chapter,` `add book` and `see progress` text are all aligned in same line please kindly fix that.
- Please kindly ensure that the `add book` button is aligned on the right like the zeplin design. the button is overflowing the border on the right.
your design
Screenshot (1282)_LI
Screenshot (1283)_LI
zeplin design
Screenshot (1284)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @DevaWinner 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Book.js (hunk 1 above)
Comment (DevaWinner) [reply src/components/Book.js:8]: Hello @tufoinnkuo10 ,

Thank you for the review. I must say that this is due to your screen. The design is perfect on my computer screen which is a 1920 x 1080 (16:9) screen resolution.

We weren't required to make it adaptable to different screen sizes hence I can't control how it appears on smaller screens.

Take a look at the screenshot below for the task on my screen.

Screenshot from 2023-07-29 20-59-54


Thank you.

**Review by mikethreels — APPROVED** [review]
Hi Winner, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job reaching out to the CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**